### PR TITLE
Desk phones

### PR DESCRIPTION
**Pull request by Affurit**
"UwU Who's this?"

### CODE REVIEW

File: code/game/objects/items/devices/radio/intercom.dm [hunk 1]
```diff
@@ -161,4 +161,14 @@
 
 /obj/item/device/radio/intercom/locked/confessional
 	name = "confessional intercom"
-	locked_frequency = 1480
\ No newline at end of file
+	locked_frequency = 1480
+	
+/obj/item/device/radio/intercom/locked/desk_phone
+	icon = "icons
```
Comment (kachnov) [inline code/game/objects/items/devices/radio/intercom.dm]: the fuck

**Review by kachnov — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/radio/intercom.dm (hunk 1 above)
Comment (Affurit) [reply code/game/objects/items/devices/radio/intercom.dm]: Uh... I don't know why that's like that...

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]

**Comment by Affurit** [comment]
FIXED

**Comment by JakeDaBoss** [comment]
Did you test this locally? I'll possibly have to create more phone instances, but I'd like you to test.

**Comment by Affurit** [comment]
The first and working version compiled fine. I'll get a screenshot of it working in a moment for you.

**Review by kachnov — APPROVED** [review]

**Review by JakeDaBoss — COMMENTED** [review]
Did you fix icon state yet?

**Pull request closed without merging**